### PR TITLE
feat: user register

### PR DESCRIPTION
**Pull request by wangsijie**
## Summary

implement API `POST /session/register`

## Testing

Local tested.

image

image

### CODE REVIEW

File: packages/core/src/routes/session.ts [hunk 1]
```diff
@@ -112,6 +113,48 @@ export default function sessionRoutes<T extends AnonymousRouter>(router: T, prov
     return next();
   });
 
+  router.post(
+    '/session/register',
```
Comment (gao-sun) [inline packages/core/src/routes/session.ts:117]: maybe `POST /session/user`?

Comment (xiaoyijun) [reply packages/core/src/routes/session.ts:117]: I think maybe `POST /session/user` is left to "create users by admin".
Registering contains the redirect logic and Creating users does not.

**Review by xiaoyijun — COMMENTED** [review]
LGTM

File: packages/core/src/routes/session.ts (hunk 1 above)
Comment (wangsijie) [reply packages/core/src/routes/session.ts:117]: admin don't use `session`

**Review by gao-sun — APPROVED** [review]

**Pull request merged**